src/scales/scale.logarithmic.js

```diff
@@ -134,8 +142,16 @@ module.exports = function(Chart) {
 			while (tickVal < me.max) {
 				ticks.push(tickVal);
 
-				var exp = Math.floor(helpers.log10(tickVal));
-				var significand = Math.floor(tickVal / Math.pow(10, exp)) + 1;
+                var exp;
+                var significand;
+
+                if(tickVal === 0){
+                    exp = Math.floor(helpers.log10(me.minNotZero));
+                    significand = Math.floor(me.minNotZero / Math.pow(10, exp));
```

Comment:
Is the `+1` removed here on purpose? If it's an error, then we could avoid some code duplication:

    var exp = Math.floor(helpers.log10(tickVal || me.minNotZero));
    var significand = Math.floor((tickVal || me.minNotZero) / Math.pow(10, exp)) + 1;